### PR TITLE
New Fabric network Configuration service: small and simple

### PR DESCRIPTION
**Pull request by ruslankryukov**

### CODE REVIEW

**Comment by andreevym** [comment]
where is new config file?

File: src/main/java/org/blockchain_monitoring/fly_client_spring/FlyConfigService.java [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
 import org.blockchain_monitoring.fly_client.FlyClient;
 import org.blockchain_monitoring.fly_client.api.ChannelClient;
+import org.blockchain_monitoring.fly_client.api.UserDataSource;
 import org.blockchain_monitoring.fly_client_spring.event.EventsProcessor;
 import org.blockchain_monitoring.fly_client_spring.model.config.COrderer;
 import org.blockchain_monitoring.fly_client_spring.model.config.Channel;
 import org.blockchain_monitoring.fly_client_spring.model.config.Endorser;
 import org.blockchain_monitoring.fly_client_spring.model.config.Event;
+import org.blockchain_monitoring.fly_client_spring.model.config.NetConfig;
+import org.blockchain_monitoring.fly_client_spring.model.config.Organisation;
```
Comment (toofast73) [inline src/main/java/org/blockchain_monitoring/fly_client_spring/FlyConfigService.java]: Organisation => OrganiZation

Comment (andreevym) [inline src/main/java/org/blockchain_monitoring/fly_client_spring/FlyConfigService.java]: +1 xD

Comment (andreevym) [inline src/main/java/org/blockchain_monitoring/fly_client_spring/FlyConfigService.java]: but why https://translate.google.ru/#en/ru/organisation

Comment (toofast73) [inline src/main/java/org/blockchain_monitoring/fly_client_spring/FlyConfigService.java]: Let's look at Oxford Dictionary, as a golden source:
https://en.oxforddictionaries.com/definition/organization

The version with Z is more common and gramatically correct. The version with S is also possible, but redirects to Z, as the main one.

Comment (ruslankryukov) [inline src/main/java/org/blockchain_monitoring/fly_client_spring/FlyConfigService.java]: We use old config file again

**Pull request merged**